app/assets/javascripts/components/activity/plagiarism_handler.jsx

```diff
@@ -1,64 +1,64 @@
 import React from 'react';
 import createReactClass from 'create-react-class';
-import SuspectedPlagiarismStore from '../../stores/suspected_plagiarism_store.js';
+import PropTypes from 'prop-types';
+import { connect } from "react-redux";
 import ActivityTable from './activity_table.jsx';
-import ServerActions from '../../actions/server_actions.js';
+import { fetchSuspectedPlagiarism } from '../../actions/suspected_plagiarism_actions.js';
+
+const HEADERS = [
+      { title: I18n.t('recent_activity.article_title'), key: 'title' },
+      { title: I18n.t('recent_activity.plagiarism_report'), key: 'report_url', style: { width: 165 } },
+      { title: I18n.t('recent_activity.revision_author'), key: 'username', style: { minWidth: 142 } },
+      { title: I18n.t('recent_activity.revision_datetime'), key: 'revision_datetime', style: { width: 200 } },
+    ];
+
+const NO_ACTIVITY_MESSAGE = I18n.t('recent_activity.no_plagiarism');
 
-const getState = () => {
-  return {
-    revisions: SuspectedPlagiarismStore.getRevisions(),
-    loading: true
-  };
-};
 
 const PlagiarismHandler = createReactClass({
   displayName: 'PlagiarismHandler',
 
-  mixins: [SuspectedPlagiarismStore.mixin],
-
-  getInitialState() {
-    return getState();
+  propTypes: {
+    fetchSuspectedPlagiarism: PropTypes.func,
+    revisions: PropTypes.object,
```

Comment:
This is an array, not an object. It is thowing a console warning with the development build of the assets.